src/Rules/Generate.hs

```diff
@@ -24,10 +25,10 @@ primopsSource :: FilePath
 primopsSource = "compiler/prelude/primops.txt.pp"
 
 primopsTxt :: Stage -> FilePath
-primopsTxt stage = contextDir (vanillaContext stage compiler) -/- "primops.txt"
+primopsTxt stage = buildDir (vanillaContext stage compiler) -/- "primops.txt"
```

Comment:
Why has this been moved from `contextDir` to `buildDir`?

Reply:
``` haskell
contextDir Context {..} = stageString stage -/- pkgPath package
buildDir context = contextDir context -/- "build"
```

@angerman is there a particular reason this got moved into the `build/` subdir there?

Reply:
unless it doesn't build with the `primops.txt` in the `contextDir`, I don't think so.